### PR TITLE
Add Tencent TKE cluster provisioning doc

### PR DESCRIPTION
**Pull request by guangbochen**
**Description:**
Add Tencent TKE cluster provisioning doc.

**Related Issues:**
#1019

### CODE REVIEW

**Comment by guangbochen** [comment]
@loganhz can u please review this one, thanks.

File: content/rancher/v2.x/en/cluster-provisioning/hosted-kubernetes-clusters/tke/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## Prerequisites
+
+Obtain the following information from the <a href='https://console.cloud.tencent.com' target='_blank'>Tencent Cloud Console</a>:
+
+- Cloud API SecretId.
+- Cloud API SecretKey.
+- Create the VPC and the Subnet in the required region.
+
+Notes: Make sure that the account you will be using to create the TKE cluster has the appropriate permissions, especially when the user is not a root account.
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/cluster-provisioning/hosted-kubernetes-clusters/tke/_index.md]: Can you list the permissions which Rancher required?

**Comment by guangbochen** [comment]
Updated and add `how to enable the TKE driver` in the cluster driver view

**Comment by loganhz** [comment]
Can you make it similar with https://github.com/rancher/docs/blob/f582ae60d3e5232070bfcda841687a99dad35e45/content/rancher/v2.x/en/cluster-provisioning/hosted-kubernetes-clusters/eks/_index.md , please?

I think you don't need to create a subpage in `content/rancher/v2.x/en/drivers/cluster-drivers/` for TKE.

Instead, can you add the doc for how to enable cluster in `driver content/rancher/v2.x/en/drivers/cluster-drivers/_index.md`? So others can link to this doc.

Something like
```
Objectives
1.  Give Appropriate Permissions (It's still not clear. You can link the tencent doce to tke required permissions? )
2. Create Access Key and Secret Key (Can you link the tencent doc to how to create the keys?)
3. Enable TKE Cluster Driver (Link to cluster-drivers page about how to enable driver)
4. Create the TKE Cluster
```

**Comment by deniseschannon** [comment]
@guangbochen can you update your PR so that it follows the same format as the following docs: 

http://staging.rancher.com/docs/rancher/v2.x/en/cluster-provisioning/hosted-kubernetes-clusters/ack/

**Comment by guangbochen** [comment]
@loganhz @deniseschannon updated according to the above comments, a review would be appreciated, thanks.

**Comment by loganhz** [comment]
LGTM

**Comment by loganhz** [comment]
LGTM

**Pull request merged**